lib/recase/cases/title_case.ex

```diff
@@ -1,25 +1,25 @@
 defmodule Recase.TitleCase do
-    @moduledoc """
-    Module to convert strings to `Title Case`.
-  
-    This module should not be used directly.
-  
-    Read about `Title Case` here:
-    https://en.wikipedia.org/wiki/Letter_case#Title_case
-    """
-
-    alias Recase.SnakeCase
-
-    @sep " "
-
-    @spec convert(String.t) :: String.t
-    def convert(""), do: ""
-    def convert(value) do
-      value
-      |> SnakeCase.convert
-      |> String.replace("_", @sep)
-      |> String.split()
-      |> Enum.map(&(String.capitalize(&1)))
-      |> Enum.join(@sep)
-    end
-  end
+  @moduledoc """
+  Module to convert strings to `Title Case`.
+
+  This module should not be used directly.
+
+  **NB!** at the moment has no stop words: titleizes everything
```

Comment:
What are "stop words" in this context? We need to document that. Or remove if it does not matter.

Reply:
Eh. There is a link to Wiki https://en.wikipedia.org/wiki/Letter_case#Title_case saying “fruits and vegetables” should be titleized as “Fruits **and** Vegetables.”

We do not support **and** remaining all lowercase. That said, we’d better off removing both warning _and_ link to Wiki.

Reply:
Sound fine, let's do it.

Reply:
Not in this PR I guess.